graphql/schemabuilder/pagination.go

```diff
@@ -686,10 +743,7 @@ func (c *connectionContext) consumeSorts(sb *schemaBuilder, m *method, typ refle
 
 		// Build a GraphQL field for the function.
 		field, err := sb.buildFunction(typ, &method{Fn: fn})
-		if err != nil {
-			return err
-		}
-		if field.Args != nil && len(field.Args) > 0 {
+		if err != nil || len(field.Args) > 0 {
```

Comment:
why did we get rid of the error return here?  If we get an error from the build func we won't see what the issue was.